app/javascript/react/components/molecules/FixedStreamStationHeader/FixedStreamStationHeader.tsx

```diff
@@ -26,7 +26,8 @@ const FixedStreamStationHeader = () => {
     endTime,
   } = useSelector(selectFixedStreamShortInfo);
 
-  const streamEndTime: string = endTime ?? lastUpdate ?? moment().format("YYYY-MM-DD");
+  const streamEndTime: string =
+    endTime ?? lastUpdate ?? moment().format("YYYY-MM-DD");
```

Comment:
We have as I remember on master now the DateType which include such "YYYY-MM-DD". Would you mind using it?